catbuffer/schemas/nem/transaction_type.cats

```diff
@@ -16,7 +16,7 @@ enum TransactionType : uint32
 	MULTISIG_COSIGNATURE = 0x1002
 
 	# multisig transaction
-	MULTISIG_TRANSACTION = 0x01004
+	MULTISIG = 0x01004
```

Comment:
hmm, not that I think any code used it, but isn't this breaking change?

Reply:
it is breaking if anyone is using the enum value directly, but the name used to create tx is unchanged as multisig_transaction_v#